src/server/kv.rs

```diff
@@ -298,7 +298,10 @@ impl StoreHandler {
         resp.set_cmd_gc_resp(gc);
     }
 
-    pub fn on_request(&self, req: Request, on_resp: OnResponse) -> Result<()> {
+    pub fn on_request(&self, mut req: Request, on_resp: OnResponse) -> Result<()> {
+        // Go client may set it to 0 if using gogoprotobuf.
+        req.mut_context().clear_term();
```

Comment:
gogoprotobuf will always set it, then we cannot use `has_term()`. See https://github.com/pingcap/kvproto/issues/97